qlty-check/src/planner/config_files.rs

```diff
@@ -93,15 +90,17 @@ pub fn plugin_configs(planner: &Planner) -> Result<HashMap<String, Vec<PluginCon
     }
 
     let mut configs: HashMap<String, Vec<PluginConfigFile>> = HashMap::new();
-    let ignore_globset = ignore_globset(&planner.config.ignore)?;
+    let exclude_globset = exclude_globset(&planner.config.exclude_patterns)?;
 
     for entry in planner.workspace.walker() {
         let entry = entry?;
         if let Some(os_str) = entry.path().file_name() {
             let file_name = os_str.to_os_string();
             for plugin_config in &plugins_configs {
+                // Why do we have exclude_globset.is_match(entry.path()) here?!
+                // If you want to exclude the file, you should not add it to the configs.
```

Comment:
We should address this comment before merging this

Reply:
I was unsure about the reasoning/actual case for this one so added a comment to bring it up.
But I suppose if it was added there might of had been a case with a user where they want to ignore configs from a certain directory.

Reply:
Actually there is a test case which covers this as well, so probably needed.